src/style.css

```diff
@@ -0,0 +1,94 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 400;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 900;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+* {
+  box-sizing: border-box;
+  color: #3d4e61;
+}
+
+.bigsearch {
+  margin-top: 20px;
+  position: relative;
+  height: 70px;
+  width: 100%;
+}
+
+.bigpicture {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  left: 26px;
+  top: 25px;
+}
+
+.biginput {
+  padding-left: 62px;
+  background: #fff;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  display: block;
+  width: 100%;
+  height: 100%;
+  border-radius: 4px;
+  font-size: 16px;
+  line-height: 22px;
+  font-family: Avenir, sans-serif;
+  border: 1px solid #e1e7ed;
+}
+
+.biginput:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.biginput:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  font-weight: 900;
+}
+
+.smallsearch {
+  margin-top: 20px;
+  position: relative;
```

Comment:
Common styles for `.bigsearch` and `.smallsearch` can be combined under some common class, like `.search` and written just once.
Same for images and inputs.